Error when installing project dependencies during opencollective-postinstall

# This is a Bug Report 

#### :tophat: Description

* What went wrong?
When installing dependencies (npm install), got an error during opencollective-postinstall
* What did you expect should have happened?
I didn't expect to get an error when installing project dependencies
* What was the config you used?
Windows 10 64 bits
npm version: '5.6.0'
* What stacktrace or error message from your provider did you see?
983 verbose stack Error: opencollective-postinstall@1.1.1 postinstall: `node bin/setup.js`
983 verbose stack Exit status 1
983 verbose stack     at EventEmitter.<anonymous> (C:\Users\Thom\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\index.js:285:16)
983 verbose stack     at emitTwo (events.js:106:13)
983 verbose stack     at EventEmitter.emit (events.js:191:7)
983 verbose stack     at ChildProcess.<anonymous> (C:\Users\Thom\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
983 verbose stack     at emitTwo (events.js:106:13)
983 verbose stack     at ChildProcess.emit (events.js:191:7)
983 verbose stack     at maybeClose (internal/child_process.js:920:16)
983 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:230:5)

#### :pushpin: Related issues

#### :clipboard: Additional Data
[2018-01-25T07_36_11_435Z-debug.log](https://github.com/DemocracyEarth/sovereign/files/1662937/2018-01-25T07_36_11_435Z-debug.log)

* ***Sovereign deployment where you found the issue***:
* ***Browser & version***:
* ***Screenshot***:
* ***Error messages***:
* ***URL to reproduce the error***:

I have the same error.
```
james@james-Streacom:~/sovereign$  meteor npm install

> bcrypt@1.0.3 install /home/james/sovereign/node_modules/bcrypt
> node-pre-gyp install --fallback-to-build

[bcrypt] Success: "/home/james/sovereign/node_modules/bcrypt/lib/binding/bcrypt_lib.node" is installed via remote

> opencollective-postinstall@1.1.1 postinstall /home/james/sovereign/node_modules/opencollective-postinstall
> node bin/setup.js

npm ERR! code ELIFECYCLE
npm ERR! errno 130
npm ERR! opencollective-postinstall@1.1.1 postinstall: `node bin/setup.js`
npm ERR! Exit status 130
npm ERR! 
npm ERR! Failed at the opencollective-postinstall@1.1.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/james/.npm/_logs/2018-02-09T05_11_44_387Z-debug.log
```
Full history of commands:
```
james@james-Streacom:~$ curl https://install.meteor.com/ | sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  7784    0  7784    0     0   1556      0 --:--:--  0:00:05 --:--:--  1375
Downloading Meteor distribution
######################################################################## 100.0%

Meteor 1.6.1 has been installed in your home directory (~/.meteor).
Writing a launcher script to /usr/local/bin/meteor for your convenience.
This may prompt for your password.
[sudo] password for james: 

To get started fast:

  $ meteor create ~/my_cool_app
  $ cd ~/my_cool_app
  $ meteor

Or see the docs at:

  docs.meteor.com

james@james-Streacom:~$ git clone https://github.com/DemocracyEarth/sovereign.git
Cloning into 'sovereign'...
remote: Counting objects: 19754, done.
remote: Compressing objects: 100% (138/138), done.
remote: Total 19754 (delta 76), reused 145 (delta 57), pack-reused 19559
Receiving objects: 100% (19754/19754), 25.21 MiB | 715.00 KiB/s, done.
Resolving deltas: 100% (11024/11024), done.
james@james-Streacom:~$ npm install
The program 'npm' is currently not installed. You can install it by typing:
sudo apt install npm
james@james-Streacom:~$ cd sovereign
james@james-Streacom:~/sovereign$ sudo apt npm install
E: Invalid operation npm
james@james-Streacom:~/sovereign$ sudo apt install npm
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages were automatically installed and are no longer required:
  linux-headers-4.13.0-24 linux-headers-4.13.0-24-generic
  linux-headers-4.13.0-29 linux-headers-4.13.0-29-generic
  linux-headers-4.13.0-30 linux-headers-4.13.0-30-generic
  linux-image-4.13.0-24-generic linux-image-4.13.0-29-generic
  linux-image-4.13.0-30-generic linux-image-extra-4.13.0-24-generic
  linux-image-extra-4.13.0-25-generic linux-image-extra-4.13.0-29-generic
  linux-image-extra-4.13.0-30-generic linux-signed-image-4.13.0-29-generic
  linux-signed-image-4.13.0-30-generic shim
Use 'sudo apt autoremove' to remove them.
The following additional packages will be installed:
  gyp libjs-inherits libjs-node-uuid libssl1.0-dev libuv1-dev node-abbrev
  node-ansi node-ansi-color-table node-archy node-async node-balanced-match
  node-block-stream node-brace-expansion node-builtin-modules
  node-combined-stream node-concat-map node-cookie-jar node-delayed-stream
  node-forever-agent node-form-data node-fs.realpath node-fstream
  node-fstream-ignore node-github-url-from-git node-glob node-graceful-fs
  node-gyp node-hosted-git-info node-inflight node-inherits node-ini
  node-is-builtin-module node-isexe node-json-stringify-safe node-lockfile
  node-lru-cache node-mime node-minimatch node-mkdirp node-mute-stream
  node-node-uuid node-nopt node-normalize-package-data node-npmlog node-once
  node-osenv node-path-is-absolute node-pseudomap node-qs node-read
  node-read-package-json node-request node-retry node-rimraf node-semver
  node-sha node-slide node-spdx-correct node-spdx-expression-parse
  node-spdx-license-ids node-tar node-tunnel-agent node-underscore
  node-validate-npm-package-license node-which node-wrappy node-yallist nodejs
  nodejs-dev nodejs-doc
Suggested packages:
  node-hawk node-aws-sign node-oauth-sign node-http-signature
The following NEW packages will be installed
  gyp libjs-inherits libjs-node-uuid libssl1.0-dev libuv1-dev node-abbrev
  node-ansi node-ansi-color-table node-archy node-async node-balanced-match
  node-block-stream node-brace-expansion node-builtin-modules
  node-combined-stream node-concat-map node-cookie-jar node-delayed-stream
  node-forever-agent node-form-data node-fs.realpath node-fstream
  node-fstream-ignore node-github-url-from-git node-glob node-graceful-fs
  node-gyp node-hosted-git-info node-inflight node-inherits node-ini
  node-is-builtin-module node-isexe node-json-stringify-safe node-lockfile
  node-lru-cache node-mime node-minimatch node-mkdirp node-mute-stream
  node-node-uuid node-nopt node-normalize-package-data node-npmlog node-once
  node-osenv node-path-is-absolute node-pseudomap node-qs node-read
  node-read-package-json node-request node-retry node-rimraf node-semver
  node-sha node-slide node-spdx-correct node-spdx-expression-parse
  node-spdx-license-ids node-tar node-tunnel-agent node-underscore
  node-validate-npm-package-license node-which node-wrappy node-yallist nodejs
  nodejs-dev nodejs-doc npm
0 to upgrade, 71 to newly install, 0 to remove and 0 not to upgrade.
Need to get 6,987 kB of archives.
After this operation, 35.5 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 gyp all 0.1+20150913git1f374df9-1ubuntu1 [265 kB]
Get:2 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 libjs-node-uuid all 1.4.0-1 [11.1 kB]
Get:3 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 nodejs amd64 6.11.4~dfsg-1ubuntu1 [4,058 kB]
Get:4 http://archive.ubuntu.com/ubuntu artful/main amd64 libuv1-dev amd64 1.9.1-3 [77.7 kB]
Get:5 http://security.ubuntu.com/ubuntu artful-security/main amd64 libssl1.0-dev amd64 1.0.2g-1ubuntu13.3 [3,236 B]
Get:6 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-async all 0.8.0-1 [22.2 kB]
Get:7 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-builtin-modules all 1.1.1-1 [3,338 B]
Get:8 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-fs.realpath all 1.0.0-1 [5,572 B]
Get:9 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-hosted-git-info all 2.1.5-1 [5,738 B]
Get:10 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-wrappy all 1.0.2-1 [3,162 B]
Get:11 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-once all 1.4.0-2 [3,398 B]
Get:12 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-inflight all 1.0.6-1 [3,382 B]
Get:13 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-is-builtin-module all 1.0.0-1 [2,906 B]
Get:14 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-isexe all 1.1.2-1ubuntu1 [4,118 B]
Get:15 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-node-uuid all 1.4.0-1 [2,530 B]
Get:16 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-path-is-absolute all 1.0.0-1 [3,310 B]
Get:17 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-pseudomap all 1.0.2-1 [3,534 B]
Get:18 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-spdx-license-ids all 1.2.2-1 [4,792 B]
Get:19 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-spdx-correct all 1.0.2-1 [3,718 B]
Get:20 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-spdx-expression-parse all 1.0.4-1 [12.1 kB]
Get:21 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-underscore all 1.8.3~dfsg-1 [3,790 B]
Get:22 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-validate-npm-package-license all 3.0.1-1 [3,488 B]
Get:23 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-yallist all 2.0.0-1 [5,398 B]
Get:24 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 nodejs-doc all 6.11.4~dfsg-1ubuntu1 [272 kB]
Get:25 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 libjs-inherits all 2.0.3-1 [2,792 B]
Get:26 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-abbrev all 1.0.9-1 [3,708 B]
Get:27 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-ansi all 0.3.0-2 [8,590 B]
Get:28 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-ansi-color-table all 1.0.0-1 [4,478 B]
Get:29 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-archy all 1.0.0-1 [4,084 B]
Get:30 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-balanced-match all 0.4.2-1 [4,030 B]
Get:31 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-inherits all 2.0.3-1 [3,092 B]
Get:32 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-block-stream all 0.0.9-1 [4,524 B]
Get:33 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-concat-map all 0.0.1-1 [3,502 B]
Get:34 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-brace-expansion all 1.1.8-1 [5,840 B]
Get:35 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-delayed-stream all 0.0.5-1 [4,750 B]
Get:36 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-combined-stream all 0.0.5-1 [4,958 B]
Get:37 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-cookie-jar all 0.3.1-1 [3,746 B]
Get:38 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-forever-agent all 0.5.1-1 [3,194 B]
Get:39 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-mime all 1.3.4-1 [11.9 kB]
Get:40 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-form-data all 0.1.0-1 [6,412 B]
Get:41 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-minimatch all 3.0.4-1 [13.0 kB]
Get:42 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-glob all 7.1.2-4 [17.7 kB]
Get:43 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-rimraf all 2.5.4-3 [7,584 B]
Get:44 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-mkdirp all 0.5.0-1 [4,690 B]
Get:45 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-graceful-fs all 4.1.11-1 [10.8 kB]
Get:46 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-fstream all 1.0.10-1 [18.1 kB]
Get:47 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-fstream-ignore all 0.0.6-2 [5,586 B]
Get:48 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-github-url-from-git all 1.4.0-1 [3,782 B]
Get:49 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 nodejs-dev amd64 6.11.4~dfsg-1ubuntu1 [278 kB]
Get:50 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-nopt all 3.0.6-3 [9,572 B]
Get:51 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-npmlog all 0.0.4-1 [5,844 B]
Get:52 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-osenv all 0.1.0-1 [3,772 B]
Get:53 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-tunnel-agent all 0.3.1-1 [4,018 B]
Get:54 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-json-stringify-safe all 5.0.0-1 [3,544 B]
Get:55 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-qs all 2.2.4-1 [7,574 B]
Get:56 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-request all 2.26.1-1 [14.5 kB]
Get:57 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-semver all 5.3.0-1 [21.9 kB]
Get:58 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-tar all 2.2.1-1 [17.7 kB]
Get:59 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-which all 1.2.11-1 [3,918 B]
Get:60 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-gyp all 3.4.0-1 [25.7 kB]
Get:61 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-ini all 1.1.0-1 [4,770 B]
Get:62 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-lockfile all 0.4.1-1 [5,450 B]
Get:63 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-lru-cache all 4.0.2-1 [7,448 B]
Get:64 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-mute-stream all 0.0.7-1 [4,372 B]
Get:65 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-normalize-package-data all 2.3.5-2 [10.6 kB]
Get:66 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-read all 1.0.7-1 [4,572 B]
Get:67 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-read-package-json all 1.2.4-1 [7,780 B]
Get:68 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-retry all 0.6.0-1 [6,172 B]
Get:69 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-sha all 1.2.3-1 [4,272 B]
Get:70 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 node-slide all 1.1.4-1 [6,118 B]
Get:71 http://au.archive.ubuntu.com/ubuntu artful/universe amd64 npm all 3.5.2-0ubuntu4 [1,586 kB]
Fetched 6,987 kB in 6s (1,055 kB/s)                                            
Extract templates from packages: 100%
Selecting previously unselected package gyp.
(Reading database ... 467207 files and directories currently installed.)
Preparing to unpack .../00-gyp_0.1+20150913git1f374df9-1ubuntu1_all.deb ...
Unpacking gyp (0.1+20150913git1f374df9-1ubuntu1) ...
Selecting previously unselected package libjs-node-uuid.
Preparing to unpack .../01-libjs-node-uuid_1.4.0-1_all.deb ...
Unpacking libjs-node-uuid (1.4.0-1) ...
Selecting previously unselected package libssl1.0-dev:amd64.
Preparing to unpack .../02-libssl1.0-dev_1.0.2g-1ubuntu13.3_amd64.deb ...
Unpacking libssl1.0-dev:amd64 (1.0.2g-1ubuntu13.3) ...
Selecting previously unselected package libuv1-dev:amd64.
Preparing to unpack .../03-libuv1-dev_1.9.1-3_amd64.deb ...
Unpacking libuv1-dev:amd64 (1.9.1-3) ...
Selecting previously unselected package nodejs.
Preparing to unpack .../04-nodejs_6.11.4~dfsg-1ubuntu1_amd64.deb ...
Unpacking nodejs (6.11.4~dfsg-1ubuntu1) ...
Selecting previously unselected package node-async.
Preparing to unpack .../05-node-async_0.8.0-1_all.deb ...
Unpacking node-async (0.8.0-1) ...
Selecting previously unselected package node-builtin-modules.
Preparing to unpack .../06-node-builtin-modules_1.1.1-1_all.deb ...
Unpacking node-builtin-modules (1.1.1-1) ...
Selecting previously unselected package node-fs.realpath.
Preparing to unpack .../07-node-fs.realpath_1.0.0-1_all.deb ...
Unpacking node-fs.realpath (1.0.0-1) ...
Selecting previously unselected package node-hosted-git-info.
Preparing to unpack .../08-node-hosted-git-info_2.1.5-1_all.deb ...
Unpacking node-hosted-git-info (2.1.5-1) ...
Selecting previously unselected package node-wrappy.
Preparing to unpack .../09-node-wrappy_1.0.2-1_all.deb ...
Unpacking node-wrappy (1.0.2-1) ...
Selecting previously unselected package node-once.
Preparing to unpack .../10-node-once_1.4.0-2_all.deb ...
Unpacking node-once (1.4.0-2) ...
Selecting previously unselected package node-inflight.
Preparing to unpack .../11-node-inflight_1.0.6-1_all.deb ...
Unpacking node-inflight (1.0.6-1) ...
Selecting previously unselected package node-is-builtin-module.
Preparing to unpack .../12-node-is-builtin-module_1.0.0-1_all.deb ...
Unpacking node-is-builtin-module (1.0.0-1) ...
Selecting previously unselected package node-isexe.
Preparing to unpack .../13-node-isexe_1.1.2-1ubuntu1_all.deb ...
Unpacking node-isexe (1.1.2-1ubuntu1) ...
Selecting previously unselected package node-node-uuid.
Preparing to unpack .../14-node-node-uuid_1.4.0-1_all.deb ...
Unpacking node-node-uuid (1.4.0-1) ...
Selecting previously unselected package node-path-is-absolute.
Preparing to unpack .../15-node-path-is-absolute_1.0.0-1_all.deb ...
Unpacking node-path-is-absolute (1.0.0-1) ...
Selecting previously unselected package node-pseudomap.
Preparing to unpack .../16-node-pseudomap_1.0.2-1_all.deb ...
Unpacking node-pseudomap (1.0.2-1) ...
Selecting previously unselected package node-spdx-license-ids.
Preparing to unpack .../17-node-spdx-license-ids_1.2.2-1_all.deb ...
Unpacking node-spdx-license-ids (1.2.2-1) ...
Selecting previously unselected package node-spdx-correct.
Preparing to unpack .../18-node-spdx-correct_1.0.2-1_all.deb ...
Unpacking node-spdx-correct (1.0.2-1) ...
Selecting previously unselected package node-spdx-expression-parse.
Preparing to unpack .../19-node-spdx-expression-parse_1.0.4-1_all.deb ...
Unpacking node-spdx-expression-parse (1.0.4-1) ...
Selecting previously unselected package node-underscore.
Preparing to unpack .../20-node-underscore_1.8.3~dfsg-1_all.deb ...
Unpacking node-underscore (1.8.3~dfsg-1) ...
Selecting previously unselected package node-validate-npm-package-license.
Preparing to unpack .../21-node-validate-npm-package-license_3.0.1-1_all.deb ...
Unpacking node-validate-npm-package-license (3.0.1-1) ...
Selecting previously unselected package node-yallist.
Preparing to unpack .../22-node-yallist_2.0.0-1_all.deb ...
Unpacking node-yallist (2.0.0-1) ...
Selecting previously unselected package nodejs-doc.
Preparing to unpack .../23-nodejs-doc_6.11.4~dfsg-1ubuntu1_all.deb ...
Unpacking nodejs-doc (6.11.4~dfsg-1ubuntu1) ...
Selecting previously unselected package libjs-inherits.
Preparing to unpack .../24-libjs-inherits_2.0.3-1_all.deb ...
Unpacking libjs-inherits (2.0.3-1) ...
Selecting previously unselected package node-abbrev.
Preparing to unpack .../25-node-abbrev_1.0.9-1_all.deb ...
Unpacking node-abbrev (1.0.9-1) ...
Selecting previously unselected package node-ansi.
Preparing to unpack .../26-node-ansi_0.3.0-2_all.deb ...
Unpacking node-ansi (0.3.0-2) ...
Selecting previously unselected package node-ansi-color-table.
Preparing to unpack .../27-node-ansi-color-table_1.0.0-1_all.deb ...
Unpacking node-ansi-color-table (1.0.0-1) ...
Selecting previously unselected package node-archy.
Preparing to unpack .../28-node-archy_1.0.0-1_all.deb ...
Unpacking node-archy (1.0.0-1) ...
Selecting previously unselected package node-balanced-match.
Preparing to unpack .../29-node-balanced-match_0.4.2-1_all.deb ...
Unpacking node-balanced-match (0.4.2-1) ...
Selecting previously unselected package node-inherits.
Preparing to unpack .../30-node-inherits_2.0.3-1_all.deb ...
Unpacking node-inherits (2.0.3-1) ...
Selecting previously unselected package node-block-stream.
Preparing to unpack .../31-node-block-stream_0.0.9-1_all.deb ...
Unpacking node-block-stream (0.0.9-1) ...
Selecting previously unselected package node-concat-map.
Preparing to unpack .../32-node-concat-map_0.0.1-1_all.deb ...
Unpacking node-concat-map (0.0.1-1) ...
Selecting previously unselected package node-brace-expansion.
Preparing to unpack .../33-node-brace-expansion_1.1.8-1_all.deb ...
Unpacking node-brace-expansion (1.1.8-1) ...
Selecting previously unselected package node-delayed-stream.
Preparing to unpack .../34-node-delayed-stream_0.0.5-1_all.deb ...
Unpacking node-delayed-stream (0.0.5-1) ...
Selecting previously unselected package node-combined-stream.
Preparing to unpack .../35-node-combined-stream_0.0.5-1_all.deb ...
Unpacking node-combined-stream (0.0.5-1) ...
Selecting previously unselected package node-cookie-jar.
Preparing to unpack .../36-node-cookie-jar_0.3.1-1_all.deb ...
Unpacking node-cookie-jar (0.3.1-1) ...
Selecting previously unselected package node-forever-agent.
Preparing to unpack .../37-node-forever-agent_0.5.1-1_all.deb ...
Unpacking node-forever-agent (0.5.1-1) ...
Selecting previously unselected package node-mime.
Preparing to unpack .../38-node-mime_1.3.4-1_all.deb ...
Unpacking node-mime (1.3.4-1) ...
Selecting previously unselected package node-form-data.
Preparing to unpack .../39-node-form-data_0.1.0-1_all.deb ...
Unpacking node-form-data (0.1.0-1) ...
Selecting previously unselected package node-minimatch.
Preparing to unpack .../40-node-minimatch_3.0.4-1_all.deb ...
Unpacking node-minimatch (3.0.4-1) ...
Selecting previously unselected package node-glob.
Preparing to unpack .../41-node-glob_7.1.2-4_all.deb ...
Unpacking node-glob (7.1.2-4) ...
Selecting previously unselected package node-rimraf.
Preparing to unpack .../42-node-rimraf_2.5.4-3_all.deb ...
Unpacking node-rimraf (2.5.4-3) ...
Selecting previously unselected package node-mkdirp.
Preparing to unpack .../43-node-mkdirp_0.5.0-1_all.deb ...
Unpacking node-mkdirp (0.5.0-1) ...
Selecting previously unselected package node-graceful-fs.
Preparing to unpack .../44-node-graceful-fs_4.1.11-1_all.deb ...
Unpacking node-graceful-fs (4.1.11-1) ...
Selecting previously unselected package node-fstream.
Preparing to unpack .../45-node-fstream_1.0.10-1_all.deb ...
Unpacking node-fstream (1.0.10-1) ...
Selecting previously unselected package node-fstream-ignore.
Preparing to unpack .../46-node-fstream-ignore_0.0.6-2_all.deb ...
Unpacking node-fstream-ignore (0.0.6-2) ...
Selecting previously unselected package node-github-url-from-git.
Preparing to unpack .../47-node-github-url-from-git_1.4.0-1_all.deb ...
Unpacking node-github-url-from-git (1.4.0-1) ...
Selecting previously unselected package nodejs-dev.
Preparing to unpack .../48-nodejs-dev_6.11.4~dfsg-1ubuntu1_amd64.deb ...
Unpacking nodejs-dev (6.11.4~dfsg-1ubuntu1) ...
Selecting previously unselected package node-nopt.
Preparing to unpack .../49-node-nopt_3.0.6-3_all.deb ...
Unpacking node-nopt (3.0.6-3) ...
Selecting previously unselected package node-npmlog.
Preparing to unpack .../50-node-npmlog_0.0.4-1_all.deb ...
Unpacking node-npmlog (0.0.4-1) ...
Selecting previously unselected package node-osenv.
Preparing to unpack .../51-node-osenv_0.1.0-1_all.deb ...
Unpacking node-osenv (0.1.0-1) ...
Selecting previously unselected package node-tunnel-agent.
Preparing to unpack .../52-node-tunnel-agent_0.3.1-1_all.deb ...
Unpacking node-tunnel-agent (0.3.1-1) ...
Selecting previously unselected package node-json-stringify-safe.
Preparing to unpack .../53-node-json-stringify-safe_5.0.0-1_all.deb ...
Unpacking node-json-stringify-safe (5.0.0-1) ...
Selecting previously unselected package node-qs.
Preparing to unpack .../54-node-qs_2.2.4-1_all.deb ...
Unpacking node-qs (2.2.4-1) ...
Selecting previously unselected package node-request.
Preparing to unpack .../55-node-request_2.26.1-1_all.deb ...
Unpacking node-request (2.26.1-1) ...
Selecting previously unselected package node-semver.
Preparing to unpack .../56-node-semver_5.3.0-1_all.deb ...
Unpacking node-semver (5.3.0-1) ...
Selecting previously unselected package node-tar.
Preparing to unpack .../57-node-tar_2.2.1-1_all.deb ...
Unpacking node-tar (2.2.1-1) ...
Selecting previously unselected package node-which.
Preparing to unpack .../58-node-which_1.2.11-1_all.deb ...
Unpacking node-which (1.2.11-1) ...
Selecting previously unselected package node-gyp.
Preparing to unpack .../59-node-gyp_3.4.0-1_all.deb ...
Unpacking node-gyp (3.4.0-1) ...
Selecting previously unselected package node-ini.
Preparing to unpack .../60-node-ini_1.1.0-1_all.deb ...
Unpacking node-ini (1.1.0-1) ...
Selecting previously unselected package node-lockfile.
Preparing to unpack .../61-node-lockfile_0.4.1-1_all.deb ...
Unpacking node-lockfile (0.4.1-1) ...
Selecting previously unselected package node-lru-cache.
Preparing to unpack .../62-node-lru-cache_4.0.2-1_all.deb ...
Unpacking node-lru-cache (4.0.2-1) ...
Selecting previously unselected package node-mute-stream.
Preparing to unpack .../63-node-mute-stream_0.0.7-1_all.deb ...
Unpacking node-mute-stream (0.0.7-1) ...
Selecting previously unselected package node-normalize-package-data.
Preparing to unpack .../64-node-normalize-package-data_2.3.5-2_all.deb ...
Unpacking node-normalize-package-data (2.3.5-2) ...
Selecting previously unselected package node-read.
Preparing to unpack .../65-node-read_1.0.7-1_all.deb ...
Unpacking node-read (1.0.7-1) ...
Selecting previously unselected package node-read-package-json.
Preparing to unpack .../66-node-read-package-json_1.2.4-1_all.deb ...
Unpacking node-read-package-json (1.2.4-1) ...
Selecting previously unselected package node-retry.
Preparing to unpack .../67-node-retry_0.6.0-1_all.deb ...
Unpacking node-retry (0.6.0-1) ...
Selecting previously unselected package node-sha.
Preparing to unpack .../68-node-sha_1.2.3-1_all.deb ...
Unpacking node-sha (1.2.3-1) ...
Selecting previously unselected package node-slide.
Preparing to unpack .../69-node-slide_1.1.4-1_all.deb ...
Unpacking node-slide (1.1.4-1) ...
Selecting previously unselected package npm.
Preparing to unpack .../70-npm_3.5.2-0ubuntu4_all.deb ...
Unpacking npm (3.5.2-0ubuntu4) ...
Setting up nodejs-doc (6.11.4~dfsg-1ubuntu1) ...
Setting up libjs-node-uuid (1.4.0-1) ...
Setting up libjs-inherits (2.0.3-1) ...
Setting up nodejs (6.11.4~dfsg-1ubuntu1) ...
update-alternatives: using /usr/bin/nodejs to provide /usr/bin/js (js) in auto mode
Setting up node-balanced-match (0.4.2-1) ...
Setting up node-node-uuid (1.4.0-1) ...
Setting up node-yallist (2.0.0-1) ...
Setting up node-slide (1.1.4-1) ...
Setting up node-github-url-from-git (1.4.0-1) ...
Setting up node-pseudomap (1.0.2-1) ...
Setting up libssl1.0-dev:amd64 (1.0.2g-1ubuntu13.3) ...
Setting up node-spdx-license-ids (1.2.2-1) ...
Setting up node-wrappy (1.0.2-1) ...
Setting up node-mime (1.3.4-1) ...
Setting up node-abbrev (1.0.9-1) ...
Setting up node-semver (5.3.0-1) ...
Setting up node-retry (0.6.0-1) ...
Setting up node-forever-agent (0.5.1-1) ...
Setting up node-underscore (1.8.3~dfsg-1) ...
Setting up gyp (0.1+20150913git1f374df9-1ubuntu1) ...
Setting up node-json-stringify-safe (5.0.0-1) ...
Processing triggers for doc-base (0.10.7) ...
Processing 4 added doc-base files...
Setting up node-inherits (2.0.3-1) ...
Setting up node-graceful-fs (4.1.11-1) ...
Setting up node-archy (1.0.0-1) ...
Setting up node-path-is-absolute (1.0.0-1) ...
Setting up node-builtin-modules (1.1.1-1) ...
Processing triggers for man-db (2.7.6.1-2) ...
Setting up node-isexe (1.1.2-1ubuntu1) ...
Setting up node-spdx-correct (1.0.2-1) ...
Setting up node-async (0.8.0-1) ...
Setting up node-cookie-jar (0.3.1-1) ...
Setting up node-mute-stream (0.0.7-1) ...
Setting up node-concat-map (0.0.1-1) ...
Setting up node-ini (1.1.0-1) ...
Setting up node-mkdirp (0.5.0-1) ...
Setting up node-once (1.4.0-2) ...
Setting up node-sha (1.2.3-1) ...
Setting up node-fs.realpath (1.0.0-1) ...
Setting up libuv1-dev:amd64 (1.9.1-3) ...
Setting up node-lockfile (0.4.1-1) ...
Setting up node-brace-expansion (1.1.8-1) ...
Setting up node-spdx-expression-parse (1.0.4-1) ...
Setting up node-qs (2.2.4-1) ...
Setting up node-osenv (0.1.0-1) ...
Setting up node-ansi (0.3.0-2) ...
Setting up node-is-builtin-module (1.0.0-1) ...
Setting up node-hosted-git-info (2.1.5-1) ...
Setting up node-delayed-stream (0.0.5-1) ...
Setting up node-tunnel-agent (0.3.1-1) ...
Setting up node-nopt (3.0.6-3) ...
Setting up node-which (1.2.11-1) ...
Setting up node-lru-cache (4.0.2-1) ...
Setting up node-combined-stream (0.0.5-1) ...
Setting up node-block-stream (0.0.9-1) ...
Setting up node-validate-npm-package-license (3.0.1-1) ...
Setting up node-inflight (1.0.6-1) ...
Setting up node-read (1.0.7-1) ...
Setting up node-form-data (0.1.0-1) ...
Setting up node-request (2.26.1-1) ...
Setting up node-minimatch (3.0.4-1) ...
Setting up nodejs-dev (6.11.4~dfsg-1ubuntu1) ...
Setting up node-normalize-package-data (2.3.5-2) ...
Setting up node-ansi-color-table (1.0.0-1) ...
Setting up node-npmlog (0.0.4-1) ...
Setting up node-glob (7.1.2-4) ...
Setting up node-rimraf (2.5.4-3) ...
Setting up node-read-package-json (1.2.4-1) ...
Setting up node-fstream (1.0.10-1) ...
Setting up node-fstream-ignore (0.0.6-2) ...
Setting up node-tar (2.2.1-1) ...
Setting up node-gyp (3.4.0-1) ...
Setting up npm (3.5.2-0ubuntu4) ...
james@james-Streacom:~/sovereign$ meteor npm run start:dev

> Sovereign.app@ start:dev /home/james/sovereign
> meteor --settings=config/development/settings.json

[[[[[ ~/sovereign ]]]]]                       

=> Started proxy.                             
=> Started MongoDB.                                                             
                                                                                
Unable to resolve some modules:

  "@babel/runtime/helpers/typeof" in /home/james/sovereign/lib/sugar.js
(web.browser)
  "@babel/runtime/helpers/inheritsLoose" in   
/home/james/sovereign/imports/ui/templates/widgets/warning/Warning.jsx
(web.browser)
  "react" in                                  
/home/james/sovereign/imports/ui/templates/widgets/warning/Warning.jsx
(web.browser)
  "@babel/runtime/helpers/assertThisInitialized" in
/home/james/sovereign/imports/ui/templates/components/identity/login/EmailLogin.jsx
(web.browser)
  "@babel/runtime/regenerator" in             
/home/james/sovereign/imports/ui/templates/components/identity/signup/Signup.jsx
(web.browser)
  "react-fastclick" in                        
/home/james/sovereign/imports/startup/client/subscribe.js (web.browser)
  "blockstack" in                             
/home/james/sovereign/imports/startup/both/modules/blockstack.js (web.browser)
  "jsontokens" in                             
/home/james/sovereign/imports/startup/both/modules/blockstack.js (web.browser)
                                              
If you notice problems related to these missing modules, consider running:
                                              
  meteor npm install --save @babel/runtime react react-fastclick blockstack
jsontokens
                                              
                                              
Unable to resolve some modules:

  "lodash" in /home/james/sovereign/imports/startup/server/accounts/accounts.js
(os.linux.x86_64)
  "blockstack" in                             
/home/james/sovereign/imports/startup/both/modules/blockstack.js
(os.linux.x86_64)
  "jsontokens" in                             
/home/james/sovereign/imports/startup/both/modules/blockstack.js
(os.linux.x86_64)
                                              
If you notice problems related to these missing modules, consider running:
                                              
  meteor npm install --save lodash blockstack jsontokens
                                              
W20180209-16:09:46.432(11)? (STDERR) /home/james/sovereign/.meteor/local/build/programs/server/boot.js:475
W20180209-16:09:46.826(11)? (STDERR) }).run();
W20180209-16:09:46.827(11)? (STDERR)    ^
W20180209-16:09:46.827(11)? (STDERR) 
W20180209-16:09:46.827(11)? (STDERR) Error: The @babel/runtime npm package could not be found in your node_modules 
W20180209-16:09:46.828(11)? (STDERR) directory. Please run the following command to install it:
W20180209-16:09:46.828(11)? (STDERR) 
W20180209-16:09:46.828(11)? (STDERR)   meteor npm install --save @babel/runtime
W20180209-16:09:46.828(11)? (STDERR) 
W20180209-16:09:46.829(11)? (STDERR)     at babel-runtime.js (packages/babel-runtime.js:25:9)
W20180209-16:09:46.829(11)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:343:9)
W20180209-16:09:46.829(11)? (STDERR)     at require (packages/modules-runtime.js:238:16)
W20180209-16:09:46.829(11)? (STDERR)     at packages/babel-runtime.js:132:15
W20180209-16:09:46.829(11)? (STDERR)     at packages/babel-runtime.js:139:3
W20180209-16:09:46.830(11)? (STDERR)     at /home/james/sovereign/.meteor/local/build/programs/server/boot.js:411:36
W20180209-16:09:46.830(11)? (STDERR)     at Array.forEach (<anonymous>)
W20180209-16:09:46.830(11)? (STDERR)     at /home/james/sovereign/.meteor/local/build/programs/server/boot.js:220:19
W20180209-16:09:46.830(11)? (STDERR)     at /home/james/sovereign/.meteor/local/build/programs/server/boot.js:471:5
W20180209-16:09:46.830(11)? (STDERR)     at Function.run (/home/james/sovereign/.meteor/local/build/programs/server/profile.js:510:12)
=> Exited with code: 1
                                              
Unable to resolve some modules:

  "@babel/runtime/helpers/typeof" in /home/james/sovereign/lib/sugar.js
(web.browser)
  "@babel/runtime/helpers/inheritsLoose" in   
/home/james/sovereign/imports/ui/templates/widgets/warning/Warning.jsx
(web.browser)
  "react" in                                  
/home/james/sovereign/imports/ui/templates/widgets/warning/Warning.jsx
(web.browser)
  "@babel/runtime/helpers/assertThisInitialized" in
/home/james/sovereign/imports/ui/templates/components/identity/login/EmailLogin.jsx
(web.browser)
  "@babel/runtime/regenerator" in             
/home/james/sovereign/imports/ui/templates/components/identity/signup/Signup.jsx
(web.browser)
  "react-fastclick" in                        
/home/james/sovereign/imports/startup/client/subscribe.js (web.browser)
  "blockstack" in                             
/home/james/sovereign/imports/startup/both/modules/blockstack.js (web.browser)
  "jsontokens" in                             
/home/james/sovereign/imports/startup/both/modules/blockstack.js (web.browser)
                                              
If you notice problems related to these missing modules, consider running:
                                              
  meteor npm install --save @babel/runtime react react-fastclick blockstack
jsontokens
                                              
                                              
Unable to resolve some modules:

  "lodash" in /home/james/sovereign/imports/startup/server/accounts/accounts.js
(os.linux.x86_64)
  "blockstack" in                             
/home/james/sovereign/imports/startup/both/modules/blockstack.js
(os.linux.x86_64)
  "jsontokens" in                             
/home/james/sovereign/imports/startup/both/modules/blockstack.js
(os.linux.x86_64)
                                              
If you notice problems related to these missing modules, consider running:
                                              
  meteor npm install --save lodash blockstack jsontokens
                                              
W20180209-16:09:59.373(11)? (STDERR) /home/james/sovereign/.meteor/local/build/programs/server/boot.js:475
W20180209-16:09:59.374(11)? (STDERR) }).run();
W20180209-16:09:59.374(11)? (STDERR)    ^
W20180209-16:09:59.375(11)? (STDERR) 
W20180209-16:09:59.375(11)? (STDERR) Error: The @babel/runtime npm package could not be found in your node_modules 
W20180209-16:09:59.375(11)? (STDERR) directory. Please run the following command to install it:
W20180209-16:09:59.375(11)? (STDERR) 
W20180209-16:09:59.375(11)? (STDERR)   meteor npm install --save @babel/runtime
W20180209-16:09:59.376(11)? (STDERR) 
W20180209-16:09:59.376(11)? (STDERR)     at babel-runtime.js (packages/babel-runtime.js:25:9)
W20180209-16:09:59.376(11)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:343:9)
W20180209-16:09:59.376(11)? (STDERR)     at require (packages/modules-runtime.js:238:16)
W20180209-16:09:59.376(11)? (STDERR)     at packages/babel-runtime.js:132:15
W20180209-16:09:59.376(11)? (STDERR)     at packages/babel-runtime.js:139:3
W20180209-16:09:59.377(11)? (STDERR)     at /home/james/sovereign/.meteor/local/build/programs/server/boot.js:411:36
W20180209-16:09:59.377(11)? (STDERR)     at Array.forEach (<anonymous>)
W20180209-16:09:59.377(11)? (STDERR)     at /home/james/sovereign/.meteor/local/build/programs/server/boot.js:220:19
W20180209-16:09:59.377(11)? (STDERR)     at /home/james/sovereign/.meteor/local/build/programs/server/boot.js:471:5
W20180209-16:09:59.378(11)? (STDERR)     at Function.run (/home/james/sovereign/.meteor/local/build/programs/server/profile.js:510:12)
=> Exited with code: 1
                                              
Unable to resolve some modules:

  "@babel/runtime/helpers/typeof" in /home/james/sovereign/lib/sugar.js
(web.browser)
  "@babel/runtime/helpers/inheritsLoose" in   
/home/james/sovereign/imports/ui/templates/widgets/warning/Warning.jsx
(web.browser)
  "react" in                                  
/home/james/sovereign/imports/ui/templates/widgets/warning/Warning.jsx
(web.browser)
  "@babel/runtime/helpers/assertThisInitialized" in
/home/james/sovereign/imports/ui/templates/components/identity/login/EmailLogin.jsx
(web.browser)
  "@babel/runtime/regenerator" in             
/home/james/sovereign/imports/ui/templates/components/identity/signup/Signup.jsx
(web.browser)
  "react-fastclick" in                        
/home/james/sovereign/imports/startup/client/subscribe.js (web.browser)
  "blockstack" in                             
/home/james/sovereign/imports/startup/both/modules/blockstack.js (web.browser)
  "jsontokens" in                             
/home/james/sovereign/imports/startup/both/modules/blockstack.js (web.browser)
                                              
If you notice problems related to these missing modules, consider running:
                                              
  meteor npm install --save @babel/runtime react react-fastclick blockstack
jsontokens
                                              
                                              
Unable to resolve some modules:

  "lodash" in /home/james/sovereign/imports/startup/server/accounts/accounts.js
(os.linux.x86_64)
  "blockstack" in                             
/home/james/sovereign/imports/startup/both/modules/blockstack.js
(os.linux.x86_64)
  "jsontokens" in                             
/home/james/sovereign/imports/startup/both/modules/blockstack.js
(os.linux.x86_64)
                                              
If you notice problems related to these missing modules, consider running:
                                              
  meteor npm install --save lodash blockstack jsontokens
                                              
W20180209-16:10:07.241(11)? (STDERR) /home/james/sovereign/.meteor/local/build/programs/server/boot.js:475
W20180209-16:10:07.242(11)? (STDERR) }).run();
W20180209-16:10:07.242(11)? (STDERR)    ^
W20180209-16:10:07.242(11)? (STDERR) 
W20180209-16:10:07.242(11)? (STDERR) Error: The @babel/runtime npm package could not be found in your node_modules 
W20180209-16:10:07.243(11)? (STDERR) directory. Please run the following command to install it:
W20180209-16:10:07.243(11)? (STDERR) 
W20180209-16:10:07.243(11)? (STDERR)   meteor npm install --save @babel/runtime
W20180209-16:10:07.243(11)? (STDERR) 
W20180209-16:10:07.244(11)? (STDERR)     at babel-runtime.js (packages/babel-runtime.js:25:9)
W20180209-16:10:07.244(11)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:343:9)
W20180209-16:10:07.244(11)? (STDERR)     at require (packages/modules-runtime.js:238:16)
W20180209-16:10:07.244(11)? (STDERR)     at packages/babel-runtime.js:132:15
W20180209-16:10:07.244(11)? (STDERR)     at packages/babel-runtime.js:139:3
W20180209-16:10:07.245(11)? (STDERR)     at /home/james/sovereign/.meteor/local/build/programs/server/boot.js:411:36
W20180209-16:10:07.245(11)? (STDERR)     at Array.forEach (<anonymous>)
W20180209-16:10:07.245(11)? (STDERR)     at /home/james/sovereign/.meteor/local/build/programs/server/boot.js:220:19
W20180209-16:10:07.245(11)? (STDERR)     at /home/james/sovereign/.meteor/local/build/programs/server/boot.js:471:5
W20180209-16:10:07.245(11)? (STDERR)     at Function.run (/home/james/sovereign/.meteor/local/build/programs/server/profile.js:510:12)
=> Exited with code: 1
=> Your application is crashing. Waiting for file change.
^C
james@james-Streacom:~/sovereign$  meteor npm install

> bcrypt@1.0.3 install /home/james/sovereign/node_modules/bcrypt
> node-pre-gyp install --fallback-to-build

[bcrypt] Success: "/home/james/sovereign/node_modules/bcrypt/lib/binding/bcrypt_lib.node" is installed via remote

> opencollective-postinstall@1.1.1 postinstall /home/james/sovereign/node_modules/opencollective-postinstall
> node bin/setup.js

npm ERR! code ELIFECYCLE
npm ERR! errno 130
npm ERR! opencollective-postinstall@1.1.1 postinstall: `node bin/setup.js`
npm ERR! Exit status 130
npm ERR! 
npm ERR! Failed at the opencollective-postinstall@1.1.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/james/.npm/_logs/2018-02-09T05_11_44_387Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "start:dev": "meteor --settings=config/development/settings.json",
     "start:debug": "meteor debug --inspect-brk=5959 --settings=config/development/settings.json",
     "lint": "eslint .",
-    "postinstall": "./node_modules/.bin/opencollective-postinstall || exit 0"
+    "postinstall": "opencollective-postinstall || exit 0"
   },
   "dependencies": {
     "@babel/runtime": "^7.0.0-beta.38",
@@ -52,7 +52,7 @@
     "eslint-plugin-jsx-a11y": "^2.2.2",
     "eslint-plugin-meteor": "^4.0.0",
     "eslint-plugin-react": "^6.2.2",
-    "opencollective-postinstall": "^1.0.11",
+    "opencollective-postinstall": "^2.0.0",
     "shell-source": "^1.1.0",
     "shelljs": "^0.7.4"
   },
@@ -82,7 +82,10 @@
       "no-underscore-dangle": "off",
       "guard-for-in": "off",
       "prefer-arrow-callback": "off",
-      "func-names": ["error", "never"],
+      "func-names": [
+        "error",
+        "never"
+      ],
       "import/no-unresolved": [
         "error",
         {

